src/common/sorted-table/index.js

```diff
@@ -343,18 +448,45 @@ export default class SortedTable extends Component {
 
               const { id = i } = item
 
-              return rowLink
+              const selectionColumn = groupedActions !== undefined && groupedActions.length !== 0 && <td>
```

Comment:
Why is it still different from the other checks?

And why not using `isEmpty`?